MemoBotCore/main.py

```diff
@@ -1,61 +1,112 @@
 from dialog_bot_sdk.bot import DialogBot
 from dialog_bot_sdk import interactive_media
+from bs4 import BeautifulSoup
+from threading import Timer,Thread,Event
+import time
 import grpc
 import os
 import json
 import requests
 
-def sendPossibleActions(*messageParams):
+kBotNick = '@memobot'
```

Comment:
So, first, CamelCase is used _only_ in class names. In other cases, please use snake_case.